### ISSUE TITLE
Can't use Vue Chrome Devtools

### ISSUE THREAD

**Issue by crash13override** [laravel/horizon#148]
I'm trying to debug a separate issue about the failed jobs being retrieved but not displayed, and I was wondering why the VueJS Devtools bar for Chrome is not working with Horizon, even though it's all written in VueJS. 

Is it something you blocked on purpose? since it's an "admin" panel, I think it's safe to keep it enabled to help debugging some edge cases.

Or is there perhaps some other reason?

thanks!

**Comment by themsaid** [comment]
Assets are compiled for production inside all Horizon views.

**Comment by crash13override** [comment]
thanks for the reply @themsaid 

Is there any easy way to recompile everything for developing so that it can be debugged?

**Comment by countbytedown** [comment]
This application is very useful, but when it stops working after an update, there is really no easy method to try and fix it. The ability to load development Vue code would really go a long way in helping the community solve issues

**Comment by themsaid** [comment]
You can install a fork on your machine and use composer repository path to load the package from that fork, this way you can recompile the assets and debug.

**Comment by crash13override** [comment]
That's a bit of a shame that we have to use a fork in order to enable the Vue developer toolbar.
I mean, it's not as if the dashboard is being used by thousands of users. It's just an admin panel that doesn't need to hide anything to the user and that doesn't need to optimize the JS code for speed, so I don't see the necessity of compiling it for production, especially now that Horizon is still in Beta stage!

Anyway, @themsaid thanks for your suggested solution and for the great work you're doing!

**Comment by driesvints** [comment]
No plans to change this. We're compiling the assets for production as Horizon is meant to be shipped and ready to go. You can fork it to debug it like @themsaid suggested.
